configs/LUXMICROAIO/config.h

```diff
@@ -0,0 +1,123 @@
+/*
+ * This file is part of Betaflight.
+ *
+ * Betaflight is free software. You can redistribute this software
+ * and/or modify this software under the terms of the GNU General
+ * Public License as published by the Free Software Foundation,
+ * either version 3 of the License, or (at your option) any later
+ * version.
+ *
+ * Betaflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *
+ * See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public
+ * License along with this software.
+ *
+ * If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#define FC_TARGET_MCU     STM32G47X
+
+#define BOARD_NAME        LUXMICROAIO
+#define MANUFACTURER_ID   LMNR
+
+#define USE_ACC
+#define USE_BARO
+#define USE_GYRO
+#define USE_FLASH
+#define USE_MAX7456
+
+#define USE_ACC_SPI_ICM42688P
+#define USE_FLASH_W25Q128FV
+#define USE_GYRO_SPI_ICM42688P
+#define USE_SPI_GYRO
+
+#define USE_BARO_DPS310
+#define USE_BARO_BPM280
+#define USE_BARO_BPM085
+#define USE_BARO_MS5611
+#define USE_BARO_QMP6988
+#define USE_BARO_LPS
```

Comment:
Please only specify physical present barometer drivers. Two drivers should be the max allowed.

Reply:
@freasy

Reply:
You can also only define USE_BARO which includes all of the baro drivers by default.

Reply:
No this hack is not a recommended practice

Reply:
Updated the config